src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -5386,10 +5398,12 @@ public static void ToUpperOverlapping()
                 Assert.Equal(expectedDestination, s1.ToUpper(CultureInfo.CurrentCulture).ToArray());
 
                 var source = new ReadOnlySpan<char>(a, 1, 3);
-                var destination = new Span<char>(a, 3, 3);
-                Assert.Equal(source.Length, source.ToUpper(destination, CultureInfo.CurrentCulture));
-                Assert.Equal(expectedDestination, destination.ToArray());
-                Assert.Equal(expectedSource, source.ToArray());
```

Comment:
nit: Looks like `expectedSource` isn't used anymore. Cleanup the unused local variables?

Reply:
yes sorry, missed